### PR TITLE
fix display driver issues in recovery

### PR DESCRIPTION
**Pull request by petterhs**
Fixing build errors in DisplayAppRecover.cpp and recoveryLoader.cpp after new updates to display driver

### CODE REVIEW

File: src/displayapp/DisplayAppRecovery.cpp [hunk 1]
```diff
@@ -83,8 +83,7 @@ void DisplayApp::DisplayLogo(uint16_t color) {
   for(int i = 0; i < displayWidth; i++) {
     rleDecoder.DecodeNext(displayBuffer, displayWidth * bytesPerPixel);
     ulTaskNotifyTake(pdTRUE, 500);
-    lcd.BeginDrawBuffer(0, i, displayWidth, 1);
-    lcd.NextDrawBuffer(reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * bytesPerPixel);
+    lcd.DrawBuffer(0, 0, displayWidth, displayHeight, reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * 2);
```
Comment (joaquimorg) [inline src/displayapp/DisplayAppRecovery.cpp:86]: this is wrong, must be
lcd.DrawBuffer(0, i, displayWidth, 1, reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * bytesPerPixel);

File: src/displayapp/DisplayAppRecovery.cpp [hunk 2]
```diff
@@ -94,8 +93,7 @@ void DisplayApp::DisplayOtaProgress(uint8_t percent, uint16_t color) {
   for(int i = 0; i < barHeight; i++) {
     ulTaskNotifyTake(pdTRUE, 500);
     uint16_t barWidth = std::min(static_cast<float>(percent) * 2.4f, static_cast<float>(displayWidth));
-    lcd.BeginDrawBuffer(0, displayWidth - barHeight + i, barWidth, 1);
-    lcd.NextDrawBuffer(reinterpret_cast<const uint8_t *>(displayBuffer), barWidth * bytesPerPixel);
+    lcd.DrawBuffer(0, 0, displayWidth, displayHeight, reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * 2);
```
Comment (joaquimorg) [inline src/displayapp/DisplayAppRecovery.cpp:96]: lcd.DrawBuffer(0, displayWidth - barHeight + i, barWidth, 1,reinterpret_cast<const uint8_t *>(displayBuffer), barWidth * bytesPerPixel);

File: src/recoveryLoader.cpp [hunk 3]
```diff
@@ -134,8 +134,7 @@ void DisplayLogo() {
   for(int i = 0; i < displayWidth; i++) {
     rleDecoder.DecodeNext(displayBuffer, displayWidth * bytesPerPixel);
     ulTaskNotifyTake(pdTRUE, 500);
-    lcd.BeginDrawBuffer(0, i, displayWidth, 1);
-    lcd.NextDrawBuffer(reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * bytesPerPixel);
+    lcd.DrawBuffer(0, 0, displayWidth, displayHeight, reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * 2);
```
Comment (joaquimorg) [inline src/recoveryLoader.cpp:137]: lcd.DrawBuffer(0, i, displayWidth, 1, reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * bytesPerPixel);

File: src/recoveryLoader.cpp [hunk 4]
```diff
@@ -145,8 +144,7 @@ void DisplayProgressBar(uint8_t percent, uint16_t color) {
   for(int i = 0; i < barHeight; i++) {
     ulTaskNotifyTake(pdTRUE, 500);
     uint16_t barWidth = std::min(static_cast<float>(percent) * 2.4f, static_cast<float>(displayWidth));
-    lcd.BeginDrawBuffer(0, displayWidth - barHeight + i, barWidth, 1);
-    lcd.NextDrawBuffer(reinterpret_cast<const uint8_t *>(displayBuffer), barWidth * bytesPerPixel);
+    lcd.DrawBuffer(0, 0, displayWidth, displayHeight, reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * 2);
```
Comment (joaquimorg) [inline src/recoveryLoader.cpp:147]: lcd.DrawBuffer(0, displayWidth - barHeight + i, barWidth, 1, reinterpret_cast<const uint8_t *>(displayBuffer), barWidth * bytesPerPixel);

**Review by joaquimorg — CHANGES_REQUESTED** [review]
The new function joins the old ones so the parameters must be the ones that already existed.

**Comment by petterhs** [comment]
Oh thank you! I was not thorough enough with this..

**Pull request merged**